mail_autosubscribe/tests/test_mail_autosubscribe.py

```diff
@@ -97,7 +97,7 @@ def test_mail_template_default_recipients(self):
         self.mail_template.use_default_to = True
         self.mail_template.send_mail(self.order.id)
         message = self.order.message_ids[0]
-        self.assertEqual(message.partner_ids, self.partner_2 | self.partner_3)
+        self.assertEqual(message.partner_ids, self.partner_2)
```

Comment:
The fact of changing the test indicates that something is not OK.

Reply:
You are right, I was confusing the context.